guide/14-deep-learning/address_standerdization_using_model_extension.ipynb

```diff
@@ -0,0 +1,955 @@
+{
```

Comment:
<p>This function is designed to collect parameters from the user through the&nbsp;<strong>Transform Text Using Deep Learning (GeoAI)</strong>&nbsp;Tool. To connect to the GPT-3.5 deployment on Azure, we require the&nbsp;<code>API key</code>,&nbsp;<code>base URL</code>, and&nbsp;<code>deployment name</code> parameters. Since these parameters are sensitive, we will prompt the user for this information instead of hardcoding it within the file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Completed</p>